mtp_cashbook/templates/disbursements/search.html

```diff
@@ -111,123 +110,122 @@ <h1 class="heading-xlarge mtp-heading--compact">{% trans 'Payments made' %}</h1>
-                    {{ disbursement.roll_number }}
-                  {% endif %}
-                {% endif %}
-              </td>
-
-              <td>
+              <td class="mtp-status mtp-status--{{ disbursement.resolution }}">
                 {{ disbursement.resolution|format_disbursement_resolution }}
               </td>
 
-              <td>
+              <td class="mtp-history">
                 {% for log_item in disbursement.log_set %}
                   {% if log_item.action == 'created' %}
-                    {{ log_item.created|date:'d/m/Y' }}
-                    {% blocktrans trimmed with staff_name=log_item.staff_name %}
-                      Entered by {{ staff_name }}
-                    {% endblocktrans %}
+                    {% trans 'Entered' %} {{ log_item.created|date:'d/m/Y' }}<br />
+                    <small>
+                      {% blocktrans trimmed with staff_name=log_item.staff_name %}
+                        by {{ staff_name }}
+                      {% endblocktrans %}
+                    </small>
 
                   {% elif log_item.action == 'edited' %}
-                    {{ log_item.created|date:'d/m/Y' }}
-                    {% blocktrans trimmed with staff_name=log_item.staff_name %}
-                      Edited by {{ staff_name }}
-                    {% endblocktrans %}
+                    {% trans 'Edited' %} {{ log_item.created|date:'d/m/Y' }}<br />
+                    <small>
+                      {% blocktrans trimmed with staff_name=log_item.staff_name %}
+                        by {{ staff_name }}
+                      {% endblocktrans %}
+                    </small>
 
                   {% elif log_item.action == 'confirmed' %}
-                    {{ log_item.created|date:'d/m/Y' }}
-                    {% blocktrans trimmed with staff_name=log_item.staff_name %}
-                      Confirmed by {{ staff_name }}
-                    {% endblocktrans %}
-                    <br/>
+                    {% trans 'Confirmed' %} {{ log_item.created|date:'d/m/Y' }}<br />
                     <small>
-                      {% trans 'NOMIS reference' %}:
-                      {{ disbursement.nomis_transaction_id|default:'—' }}
+                      {% blocktrans trimmed with staff_name=log_item.staff_name %}
+                        by {{ staff_name }}
+                      {% endblocktrans %}
                     </small>
-
-                  {% elif log_item.action == 'sent' %}
-                    {{ log_item.created|date:'d/m/Y' }}
-                    {% blocktrans trimmed %}
-                      Sent by SSCL
-                    {% endblocktrans %}
                     <br/>
                     <small>
-                      {% trans 'Invoice number' %}:
-                      {{ disbursement.invoice_number }}
+                      {% trans 'NOMIS ref' %}:
+                      {{ disbursement.nomis_transaction_id|default:'' }}
```

Comment:
so if it's not available, you think it's best to display nothing after "NOMIS ref"?